### ISSUE TITLE
Maintainer

### ISSUE THREAD

**Issue by VQ9** [Scyntrus/FactionMobs#32]
Would like the become the maintainer of FactionsMob. I tried to update it to 1.15.2 but got stuck at the obfuscation.

**Comment by Scyntrus** [comment]
The quick and dirty thing to do is look at the source of the old version, find the function you're looking for, then open up the new version and look at approximately the same part of the file. You'll find a function that looks about the same but maybe has a different name, then just update with the new name. But you really should try to understand how it all works otherwise you'd be screwed if they release a version with significant architectural changes, which as happened several times during my development.

**Comment by SydMontague** [comment]
The [MiniMappingViewer](https://minidigger.github.io/MiniMappingViewer/) is a useful tool to deal with Mojang's obfuscation, especially since their own mapping is now public.

You can also take a look at my fork, where I added 1.15.2 compatibility. I didn't create a Pull Request since I didn't want to deal with all the dependencies. 
https://github.com/SydMontague/FactionMobs/commits/master

**Comment by Scyntrus** [comment]
Wow they released the mappings around the same time I stopped updating this mod. 

But if you remove the dependencies this mod doesn't work at all. How do you even test it?

**Comment by SydMontague** [comment]
I let my own clans plugin inject itself into FactionMobs on startup.

**Comment by Scyntrus** [comment]
That's neat, but you can't release it as a replacement for the original if it doesn't work with the same dependencies.

**Comment by SydMontague** [comment]
I didn't intend to, I just wanted to make my changes available in case someone else wants to upstream them while maintaining the dependencies.

**Comment by Scyntrus** [comment]
I see. Was there anything actionable you wanted me to do? Otherwise I'll be closing this issue.

**Comment by SydMontague** [comment]
Not really, I just wanted to provide my insight to the creator of this issue in case they still intend on maintaining the plugin in the future.

**Comment by Chronic-Reflexes** [comment]
Why dont the 3 of yall  (if your each up to it) Maintain the plugin as a goup
